src/filters/FullPoseEstimator.cpp

```diff
@@ -104,7 +106,8 @@ void FullPoseEstimator::correctHeading(double heading) {
 
 void FullPoseEstimator::predict(double thetaVel, double xVel) {
 	// convert xVel, thetaVel to wheel velocities
-	wheelvel_t wheelVels = kinematics.robotVelToWheelVel(xVel, thetaVel);
+	kinematics::wheelvel_t wheelVels =
```

Comment:
Same here: now that you specified `use namespace kinematics` on line 7, the explicit namespace specifier is unneeded